qrenderdoc/Windows/PipelineState/VulkanPipelineStateViewer.cpp

```diff
@@ -224,7 +224,7 @@ VulkanPipelineStateViewer::VulkanPipelineStateViewer(ICaptureContext &ctx,
     res->setHeader(header);
 
     res->setColumns({QString(), tr("Set"), tr("Binding"), tr("Type"), tr("Resource"),
-                     tr("Contents"), tr("cont.d"), tr("Go")});
+                     tr("Contents"), tr("Addendum"), tr("Go")});
```

Comment:
While it's correct I think 'Addendum' is less likely to be immediately understood by non-native english speakers. It's a fair point that cont.d / cont'd is not a very obvious contraction - I'd rather just expand this to 'Continued' though in that case, or perhaps 'Additional'.

Reply:
OK. I went with Additional as it felt clearer (Additional information) than Continued (information from the column on the left).